cli/src/cmd_disk.rs

```diff
@@ -76,6 +76,10 @@ pub struct CmdDiskImport {
     /// The version of this image (e.g. 11, focal, a9e77e3a, 2023-04-06T14:23:34Z)
     #[clap(long, requires_all = ["snapshot", "image", "image_description", "image_os"])]
     image_version: Option<String>,
+
+    /// The number of parallel threads to use during upload
+    #[clap(long, default_value = "8")]
+    thread_count: usize,
```

Comment:
```suggestion
    /// The degree of parallelism to use during upload
    #[clap(long, default_value = "8", hide = true)]
    parallelism: usize,
```

Let's rename this to be more accurate (even if the underlying API is a little imprecise) and let's hide it so that customer aren't surprised when we remove it.

Reply:
Done